api/src/pcapi/core/educational/adage_backends/logger.py

```diff
@@ -101,22 +101,22 @@ def get_cultural_partner(self, siret: str) -> venues_serialize.AdageCulturalPart
         logger.info("Adage has been called at %s", f"{self.base_url}/v1/etablissement-culturel/{siret}")
         if siret == "12345678200010":
             return venues_serialize.AdageCulturalPartner(
-                id="128028",
+                id="128028",  # type: ignore [arg-type]
```

Comment:
`AdageCulturalPartner.id` est un int : 

```suggestion
                id=128028,
```